src/xcode/ENA/ENA/Source/AppDelegate+ENATaskExecutionDelegate.swift

```diff
@@ -24,70 +24,79 @@ import UIKit
 extension AppDelegate: ENATaskExecutionDelegate {
 
 	/// This method executes the background tasks needed for: a) fetching test results and b) performing exposure detection requests
-	func executeENABackgroundTask(task: BGTask, completion: @escaping ((Bool) -> Void)) {
-		executeFetchTestResults(task: task) { fetchTestResultSuccess in
-
-			// NOTE: We are currently fetching the test result first, and then execute
-			// the exposure detection check. Instead of implementing this behaviour in the completion handler,
-			// queues could be used as well. Due to time/resource constraints, we settled for this option.
-			self.executeExposureDetectionRequest(task: task) { exposureDetectionSuccess in
-				self.executeFakeRequests() {
-					completion(fetchTestResultSuccess && exposureDetectionSuccess)
-				}
-			}
+	func executeENABackgroundTask(completion: @escaping ((Bool) -> Void)) {
```

Comment:
what is the purpose of this method? From the comment, I cannot figure it out. All the results are just discarded.

Reply:
Agree, the method is a bit confusing- I added a comment that explicitly states that we always call completion(true) when all subtasks finished, regardless of their individual success. In this context here, completion(true) indicates nothing but that all subtasks were able to finish.